### PR TITLE
Use Coiled Functions in arxiv matplotlib example

### PR DESCRIPTION
**Pull request by jrbourbeau**
Closes https://github.com/coiled/examples/issues/33

### CODE REVIEW

**Comment by jrbourbeau** [comment]
Note the major change here is I'm currently only processing the first 1000 directories at the moment. This takes ~3.5 minutes with Coiled Functions scaling up to 100 workers (the default adaptive limit). Doing the full 6000+ directories is probably too slow for this example. Some options would be to allow ourselves to adaptive scale to more VMs or to use the new `threads_per_worker` kwarg

**Comment by dchudz** [comment]
> Some options would be to allow ourselves to adaptive scale to more VMs or to use the new threads_per_worker kwarg

I'd go with one of those options. IMO example becomes a lot less compelling if we restrict its size to what we could easily handle. "Sorry too big" isn't really the look I want for Coiled.

**Comment by mrocklin** [comment]
Let's change the default adaptive maximum for functions.

**Comment by dchudz** [comment]
sounds good

**Comment by ntabris** [comment]
What do people think about also changing the example so that workload runs on single-core ARM workers? That's the best way I've found to run this.

(If we do this, I propose we also tweak something so that scheduler is not single-core machine in this case.)

**Comment by jrbourbeau** [comment]
Sounds good -- I've found


```python
arm=True
cpu=1
spot_policy="spot_with_fallback"
```

to be good setting for churning through lots of small files

File: arxiv-matplotlib.ipynb [hunk 1]
```diff
@@ -204,50 +227,32 @@
...
-    "\n",
-    "futures = client.map(extract, directories)\n",
-    "wait(futures)\n",
-    "\n",
-    "# We had one error in one file.  Let's just ignore and move on.\n",
-    "good = [future for future in futures if future.status == \"finished\"]\n",
-    "\n",
-    "lists = client.gather(good)"
+    "results = extract.map(directories[:1000])\n",
+    "lists = list(results)"
```
Comment (mrocklin) [inline arxiv-matplotlib.ipynb:247]: Why this subsetting?

Comment (jrbourbeau) [reply arxiv-matplotlib.ipynb:249]: That's a leftover from testing things out earlier. Just pushed a commit removing it

**Comment by jrbourbeau** [comment]
I'm going to merge this in -- happy to follow-up if folks have additional comments though

**Pull request merged**